Yank/utils.py

```diff
@@ -178,8 +178,13 @@ def initialize_mpi():
     # Override sys.excepthook to abort MPI on exception
     def mpi_excepthook(type, value, traceback):
         sys.__excepthook__(type, value, traceback)
+        sys.stdout.flush()
+        sys.stderr.flush()
         if mpicomm.size > 1:
             mpicomm.Abort(1)
+    # Store old exception handler
+    sys.__excepthook__ = sys.excepthook
```

Comment:
This line is not necessary. `__excepthook__` always stores the original: https://docs.python.org/3.5/library/sys.html#sys.__excepthook__